app/business/payments/payouts/payouts.rb

```diff
@@ -7,7 +7,7 @@ class Payouts
   PAYOUT_TYPE_STANDARD = "standard"
   PAYOUT_TYPE_INSTANT = "instant"
 
-  def self.is_user_payable(user, date, processor_type: nil, add_comment: false, from_admin: false)
+  def self.is_user_payable(user, date, processor_type: nil, add_comment: false, from_admin: false, payout_type: Payouts::PAYOUT_TYPE_STANDARD)
```

Comment:
i added instant payout validation logic to the `Payouts.is_user_payable`. 

I didn't touch `InstantPayoutsService` (which is just used for one-off instant payouts), but in the future it should probably use `Payouts.is_user_payable` so that every payout flows through the same check.